packages/html-elements/README.md

```diff
@@ -0,0 +1,493 @@
+  - The `H1` component will render an `<h1 />` on web, a `UILabel` on iOS, and a `TextView` on Android.
+- Every component excepts styles from the `StyleSheet` API.
+- TypeScript works for iOS, Android, and web, no need to monkey patch types.
+- This package is completely side-effect free!
+- Every component is tested universally for iOS, Android, and Web using the package [`jest-expo-enzyme`](https://www.npmjs.com/package/jest-expo-enzyme).
+
+## Setup
+
+Install:
+
+```sh
+yarn add @expo/html-elements
+```
+
+Import and use the package:
+
+```tsx
+import { H1 } from '@expo/html-elements';
+```
+
+# Components
+
+Not all HTML elements are supported. There are some HTML elements that mostly overlap with some universal modules, you should always try to use the universal modules whenever possible. All supported components are a capitalized variation of the semantic HTML they implement/emulate.
+
+- [x] `<h1 />` => `<H1 />`
+- [x] `<h2 />` => `<H2 />`
+- [x] `<h3 />` => `<H3 />`
+- [x] `<h4 />` => `<H4 />`
+- [x] `<h5 />` => `<H5 />`
+- [x] `<h6 />` => `<H6 />`
+- [x] `<a />` => `<A />`
+- [x] `<article />` => `<Article />`
+- [x] `<header />` => `<Header />`
+- [x] `<main />` => `<Main />`
+- [x] `<section />` => `<Section />`
+- [x] `<nav />` => `<Nav />`
+- [x] `<footer />` => `<Footer />`
+- [x] `<p />` => `<P />`
+- [x] `<b />` => `<B />`
+- [x] `<s />` => `<S />`
+- [x] `<strike />` => `<Strike />`
+- [x] `<strong />` => `<Strong />`
+- [x] `<i />` => `<I />`
+- [x] `<em />` => `<Em />`
+- [x] `<br />` => `<Br />`
+- [x] `<small />` => `<Small />`
+- [x] `<mark />` => `<Mark />`
+- [x] `<code />` => `<Code />`
+- [x] `<hr />` => `<Hr />`
+- [x] `<table />` => `<Table />`
+- [x] `<thead />` => `<Thead />`
+- [x] `<tbody />` => `<Tbody />`
+- [x] `<tfoot />` => `<Tfoot />`
+- [x] `<th />` => `<Th />`
+- [x] `<tr />` => `<Tr />`
+- [x] `<td />` => `<Td />`
+- [x] `<caption />` => `<Caption />`
+- [x] `<ul />` => `<Ul />`
+- [x] `<ol />` => `<Ol />`
+- [x] `<li />` => `<Li />` or `<FlatList />` from `react-native-web`
+- [ ] `<div />` => `<View />` from `react-native-web`
+- [ ] `<video />` => Use `<Video />` from [`expo-av`](https://docs.expo.io/versions/latest/sdk/video/).
+- [ ] `<img />` => Use `<Image />` from `react-native-web`
+- [ ] `<canvas />` => Use `<GLView />` from [`expo-gl`](https://docs.expo.io/versions/latest/sdk/gl-view/)
+- [ ] `<input type="text" />` => Use `<TextView />`
+- [ ] `<input type="file" />` => Use [`expo-image-picker`](https://docs.expo.io/versions/latest/sdk/imagepicker/) and [`expo-document-picker`](https://docs.expo.io/versions/latest/sdk/document-picker/)
+- [ ] `<iframe />` => On native use `<WebView />` from `@react-native-community/web-view`, this package is not maintained by Expo and doesn't have web support.
+
+## Headings
+
+Header elements will use the expected font size and margins from web universally.
```

Comment:
might be good to refer to exactly where this information came from, eg: http://trac.webkit.org/browser/trunk/Source/WebCore/css/html.css if that's where you found it. also would be good to explain how the default styles using ems/rems is translated to native by referring people to the css/units file